src/htm/regions/TMRegion.cpp

```diff
@@ -336,7 +343,7 @@ Spec *TMRegion::createSpec() {
           NTA_BasicType_UInt32,          // type
           1,                             // elementCount
           "",                            // constraints
-          "8",                           // defaultValue
+          "10",                          // defaultValue
```

Comment:
so the defaults now match the Algo version (?) It's good that thanks to this example, we've validated that people using NetworkAPI get the same quality out of the box :+1:

Reply:
This is a test to confirm that this field can be changed.  Could have used any field.  I had to use a value of 11 rather than 8 because it had to be larger than the minimum which is now 10.

Reply:
Yes, I changed all NetworkAPI defaults to match the Algorithm defaults.  This required a manual eye-ball compare.  We don't have a way to write a test that can check if the defaults are the same in all cases.